scripts/eurostat/regional_statistics_by_nuts/education_enrollment/education_enrollment_preprocess_gen_tmcf.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+import io
+import csv
+
+_DATA_URL = "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/trng_lfse_04.tsv.gz"
+_CLEANED_CSV = "./Eurostats_NUTS2_Enrollment.csv"
+_TMCF = "./Eurostats_NUTS2_Enrollment.tmcf"
+
+output_columns = [
```

Comment:
```suggestion
_OUTPUT_COLUMNS = [
```

Also please change the uses.  Thanks.